ruby/vampires.rb

```diff
@@ -0,0 +1,110 @@
+def age_check(age, year)
+	if 2017 - age == year

```

Comment:
Consider checking  the birth year agains the Ruby time class `Time.now.year` this makes the code more dynamic and allows you the freedom of not having to rewrite the file with hardcoded values as years change.